storage/CMakeLists.txt

```diff
@@ -1196,6 +1229,44 @@ target_link_libraries(SimpleScalarSeparateChainingHashTable_unittest
 add_test(SimpleScalarSeparateChainingHashTable_unittest
          SimpleScalarSeparateChainingHashTable_unittest)
 
+add_executable(SMAIndexSubBlock_unittest "${CMAKE_CURRENT_SOURCE_DIR}/tests/SMAIndexSubBlock_unittest.cpp")
```

Comment:
Move this part at least before `SimpleScalarSeparateChainingHashTable_unittest`.